### ISSUE TITLE
?-menu link to online User Manual should be named consistently

### ISSUE THREAD

**Issue by pryrt** [notepad-plus-plus/notepad-plus-plus#10087]
### Description of the Issue

The **?**-menu's link to the [online user manual](npp-user-manual.org) doesn't have consistent naming.  By default (per the  [Notepad_plus.rc](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/9a2dcaa5f8eb678e2b0c6494c16a8b9df37f06bb/PowerEditor/src/Notepad_plus.rc#L1135)), it is labeled "Notepad++ Online Document".  But if you use the **Settings > Preferences > General > Localization** pulldown to choose either [english.xml](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/f9d6fb9e31719a0ef9bbc8bd42c262560575adf8/PowerEditor/installer/nativeLang/english.xml#L322) or [english_customizable.xml](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/585608b456d5c7373bbae9e02a86d3e91059b775/PowerEditor/installer/nativeLang/english_customizable.xml#L319), that menu entry will shrink down to "Online Documentation" -- so even though the default is English, it doesn't match the two English localization files.

The naming should be consistent between Notepad_plus.rc, english.xml, and english_customizable.xml.  Also, because the branding of the npp-user-manual is "Notepad++ User Manual", I think it would be better to make the English name for all three locations as "Notepad++ User Manual".

### Steps to Reproduce the Issue

1. Fresh installation / unzip
2. Look at **?** menu and see "Notepad++ Online Document"
3. **Settings > Preferences > General > Localization**: select `English (customizable)`
4. Look at **?** menu and see "Online Documentation"
5. **Settings > Preferences > General > Localization**: select `English`
6. Look at **?** menu and see "Online Documentation"

### Expected Behavior

See the same text whether it's the default menu text, english.xml menu text, or english_customizable.xml menu text.  Suggested text would be "Notepad++ User Manual"

### Actual Behavior

See different text in the default menu text vs the english.xml or english_customizable.xml menu text.

### Debug Information

```
Notepad++ v8.1   (64-bit)
Build time : Jun 17 2021 - 01:56:07
Path : C:\usr\local\apps\npp\npp.8.1.portable.x64\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1903
OS Build : 18362.1256
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 
```

**Comment by sasumner** [comment]
This is a good suggestion.
I've liked the "Online" part of the existing text, because it reminds me that it will open a browser if I choose it.
But I suppose it is OK to drop that, as the 3 neighboring entries in that menu are all obviously "online" things.

**Comment by pryrt** [comment]
(weird, I thought I replied some time ago; apparently didn't hit **Comment**)

I was originally going to suggest `"Notepad++ Online User Manual" or "Notepad++ User Manual"` and allow the implementer to decide.  But since the website calls itself "Notepad++ User Manual", I went with that for the Issue.

The "Notepad++ Online User Manual" does help the user understand that it's going to be a website, rather than a link to a usermanual that was installed locally with Notepad++.  So I can see that as a good argument.  (As a lesser argument, it used to say "Online", so it should still say "Online".)

If the implementer were to choose "Notepad++ Online User Manual", I would be quite happy with the results.

But the most important part of this request (to me) is to not have a discrepancy between the default and english/english_customizable text, with a close second of using "User Manual" in some form rather than "Document" or "Documentation"
